### PR TITLE
ENT-4023 DC-to-VCS scripts

### PR DESCRIPTION
**Pull request by Lex-2008**
merge with https://github.com/cfengine/nova/pull/1247 and https://github.com/cfengine/buildscripts/pull/449

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @olehermanse can review this?

File: contrib/masterfiles-stage/README.org [hunk 1]
```diff
@@ -232,8 +212,9 @@ Or execution can be automated with CFEngine.
 #+END_SRC
 
 If the params file is placed in the default location
-=/opt/cfengine/dc-scripts/params.sh=, and if the value of =MASTERDIR=
-is specified in the params file, the CFEngine code can be as simple as:
+=/opt/cfengine/dc-scripts/params.sh=, and masterfiles to be deployed
+are in the default location =/var/cfengine/masterfiles=, the CFEngine
```
Comment (vpodzime) [inline contrib/masterfiles-stage/README.org]: I think `=MASTERDIR=` was better here. When I was trying to create the packages for Fedora, I had to use a different directory for *masterfiles*.

Comment (Lex-2008) [reply contrib/masterfiles-stage/README.org]: agreed

**Comment by Lex-2008** [comment]
requested changes implemented! @nickanderson : approve?

**Pull request merged**